test/torchaudio_unittest/prototype/hdemucs_test_impl.py

```diff
@@ -106,3 +116,115 @@ def test_decoder_output_shape_time(self):
 
         assert z.size() == (batch_size, chout, t * stride)
         assert y.size() == (batch_size, chin, t)
+
+
+@skipIfNoModule("demucs")
+class TestDemucsIntegration(TorchaudioTestCase):
+    """Test the process of importing the models from demucs.
+
+    Test methods in this test suite check the following things
+    1. Models loaded with demucs can be imported.
```

Comment:
I guess this description is from w2v2 test, but it does not apply here. w2v2 test also includes the function that converts original w2v2 to torchaudio's, which is not the case for Demucs.